front/app/api/permissions/types.ts

```diff
@@ -30,7 +30,7 @@ export interface IGlobalPermissionData {
   };
 }
 // TODO: JS - change name - PermissionContextAction?
-export type IParticipationContextPermissionAction =
+export type IPhasePermissionAction =
```

Comment:
@EdwinKato I've renamed a few types, but the same names seem to be in the cypress commands and the component library - assume I should probably change them there as well?